.eslintrc.yml

```diff
@@ -1,3 +1,7 @@
-extends: cheminfo
+extends: cheminfo-typescript
 parserOptions:
   sourceType: module
+settings:
+  import/resolver:
+    node:
+      extensions: ['.js', '.ts']
```

Comment:
why do you need this?

Reply:
Because I was getting `import/no-unresolved` error in check lint phase, So I needed to add that.

Reply:
That seems wrong. Can you tell me in what file was the error?

Reply:
Is it possible to rename the test files to `.ts` instead?

Reply:
Thank you for the review I just changed the extension of all test files and it's working now.